modules/core/domain/src/main/java/org/rhq/core/domain/content/PackageVersion.java

```diff
@@ -290,6 +292,7 @@
     public static final String QUERY_FIND_BY_ID = "PackageVersion.findById";
     public static final String QUERY_FIND_PACKAGEVERSION_BY_FILENAME = "PackageVersion.findPackageVersionByFilename";
     public static final String QUERY_FIND_DELETEABLE_IDS_IN_REPO = "PackageVersion.findDeleteableVersionIds";
+    public static final String QUERY_FIND_PACKAGE_HISTORICAL_VERSIONS = "PackageVersion.findPackageOtherVersions";
```

Comment:
Could you please use the same name for both the constant and the string content? 
Unless I'm missing something we might prefer to be consistent.